src/hooks/useInitialFocus.ts

```diff
@@ -0,0 +1,9 @@
+import { RefObject, useEffect } from 'react';
+const useInitialFocus = (ref: RefObject<HTMLInputElement>): void => {
```

Comment:
Kan legge til en liten dokumentasjon/beskrivelse til hooken.

Reply:
Anbefaler "Document This" VSCode extension

Reply:
> Kan legge til en liten dokumentasjon/beskrivelse til hooken.

Er faktisk ikke sikker med hensyn på dokumentasjon :)
Og når jeg begynner å grave litt, så er denne hooken ikke i bruk i det hele tatt...

Note: Finnes en tilsvarende hook som heter useFocus som sannsynligvis også kan dekke behovet denne gjør - bare at brukeren av hooken selv må kalle setFocus i useEffect-metoden (tipper jeg)..

Reply:
Sant nok, det er ikke den vanskeligeste hooken å forstå, men når vi legger ting til i f. eks EchoUtils og EchoComponents gjør vi det til en vane at alt skal ha en eller annen form for dokumentasjon slik at vi er helt tydelige.

La merke til det samme med useFocus og useInitialFocus.. men siden de ikke fungerer på helt samme måte tenkte jeg det kanskje finnes et use case for begge to, men litt rart hvis den faktisk ikke blir brukt noe sted hehe. useFocus kan jo kreve en form for user interaction for å sette focus, mens useInitialFocus vil sette focus on mount og så ikke gjøre noe mer.

Reply:
Forresten, når vi bruker "document this" til å dokumentere får vi disse kjekke forklaringene når vi hoverer over funksjoner/hooks 😃
![image](https://user-images.githubusercontent.com/74668394/112966435-86cb2600-914a-11eb-8446-1ab640001ecc.png)

Reply:
Vi har litt gammel kode som ikke har blitt slettet i echopedia. Hvis det ikke er i bruk, skal det som regel slettes :)